pipeline/Dockerfile

```diff
@@ -24,10 +24,10 @@ RUN wget --progress=dot:giga "https://github.com/yannh/kubeconform/releases/down
 
 # sops
 # NOTE: This needs to be installed before the helm-secrets plugin.
-ENV SOPS_VERSION="3.7.3"
-RUN wget --progress=dot:giga "https://github.com/getsops/sops/releases/download/v${SOPS_VERSION}/sops-v${SOPS_VERSION}.linux" && \
-    install -Tm 755 "sops-v${SOPS_VERSION}.linux" /usr/local/bin/sops && \
-    rm "sops-v${SOPS_VERSION}.linux"
+ENV SOPS_VERSION="3.8.1"
```

Comment:
Should we update https://github.com/elastisys/compliantkubernetes-apps/blob/main/roles/get-requirements.yaml#L13 as well?

Reply:
I don't know the impact of it for the rest of the repository so that should probably be a different task?

Reply:
I just feel that we should try to keep the pipeline as close to a dev environment as possible and in that case we'd like to keep these two versions the same. But I guess if we get a task for looking into that now then it'll soon become synced again

Reply:
Yea that is a good point. We probably should install the dependencies the same way then, to prevent this.

Reply:
I think these tests are going to be migrated to the new test framework @aarnq is working on though, so I won't create any issues until things have settled.